src/config/index.js

```diff
@@ -3,4 +3,6 @@ const registerConfig = require('windbreaker-service-util/config')
 const Config = require('./Config')
 const config = module.exports = new Config()
 
-registerConfig(config)
+registerConfig(config, [
+  { amqUrl: `amqp://localhost:${process.env.AMQ_PORT || 5672}` }
+])
```

Comment:
Yes, that's right. You're using the correct syntax for overrides. `server-config.yml` should be loaded in and used as an override here, and the `process.env.AMQ_PORT` should be used as the second override. Perhaps utility functions for building the override configs should be built into the `windbreaker-service-util`. For example:

```js
const configFile = loadConfig('../server-config.yml')
const processConfig = buildProcessConfig(process.env) // Return relevant process config args

registerConfig(config, [
  configFile, // Assuming config file is an object,
  processConfig
])
```